src/main/java/net/fabricmc/loom/providers/MappingsProvider.java

```diff
@@ -93,9 +92,8 @@ public void provide(DependencyInfo dependency, Project project, LoomGradleExtens
 
 		boolean isV2 = doesJarContainV2Mappings(mappingsJar.toPath());
 
-		Version mappingsVersion = new Version(version);
-		this.minecraftVersion = mappingsVersion.getMinecraftVersion();
-		this.mappingsVersion = mappingsVersion.getMappingsVersion() + (isV2 ? "-v2" : "");
+		this.minecraftVersion = minecraftProvider.minecraftVersion;
+		this.mappingsVersion = version + (isV2 ? "-v2" : "");
```

Comment:
Is mappingsVersion used outside of this class?

Reply:
it's used in the dependency manager and mapped mc provider

Reply:
Wont this version also contain the MC version now?

Reply:
Yes?

Reply:
The file names may contain the mc version twice now though, would it be worth removing it from this version here?

Reply:
@natanfudge has supported having two versions as he reasoned that this would allow usage of e.g. 1.14.3 mapping on 1.14.4 etc. (as what happens on mcp side)